src/js/components/Tabs/stories/Plain.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import UncontrolledTabs from './Uncontrolled';
+
+storiesOf('Tabs', module).add('Plain', () => <UncontrolledTabs plain />);
```

Comment:
I think this will be more readable if we'll place it on the same page as UncontrolledTabs, this way it will be easier to review to code for the `Plain` story on the storybook.

Reply:
Done.